### DISCUSSION TITLE
I updated LND to latest version (v0.17.0-beta) using MYNODE container, and some services including wallets are not initiating

### DISCUSSION THREAD

**Discussion by caiomsbr** [lightningnetwork/lnd#8144] [Troubleshooting]
If someone else had the same issue, please help, I'm still unable to access my balance through wallets although LND, LNDG, LNCLI are funcional, the node is operational, even forwarding payments.

LNBits, Pool, Loop, Terminal, and Ride the Lightning are unable to run...

![2023-11-01 - 13 45 03 ~ myNode Status - Google Chrome](https://github.com/lightningnetwork/lnd/assets/50489204/e72775b1-c2b4-4c58-bb03-4748c730b622)

![2023-11-01 - 13 48 30 ~ myNode Status - Google Chrome](https://github.com/lightningnetwork/lnd/assets/50489204/0e0d74bd-c218-4183-8724-933a68c2864d)

![2023-11-01 - 16 44 21 ~ myNode Status - Google Chrome](https://github.com/lightningnetwork/lnd/assets/50489204/5c78db35-168c-46d3-86e8-782aa6ebf190)

I tried rescaning LND twice, problem persists! LND version 0.16.4 was working fine!

**Comment by caiomsbr** [comment]
I can see mynode tries to get answers from synced lnd using this code:

echo "Checking if LND is synced..."
12 | lncli --network=$NETWORK --lnddir /mnt/hdd/mynode/lnd getinfo | grep 'synced_to_chain": true'
13 | while [ $? -ne 0 ]; do
14 | echo "LND not synced, sleeping for 60 seconds..."
15 | /bin/sleep 60s
16 | lncli --network=$NETWORK --lnddir /mnt/hdd/mynode/lnd getinfo | grep 'synced_to_chain": true'
17 | done

it checks using lnd getinfo, but here's what my system responds:

/home/bitcoin/.lnd/data$ lnd getinfo
failed to load config: ValidateConfig: either bitcoin.active or litecoin.active must be set to 1 (true)

In my lnd.conf the line "bitcoin.active=1" is present.

**Reply by bitromortac** [reply]
You used `lnd getinfo` instead of `lncli getinfo`, it may be that you need to give it some time to let it sync.
